tools/mason/MasonNew.chpl

```diff
@@ -152,24 +289,39 @@ proc addGitIgnore(dirName: string) {
   GIwriter.close();
 }
 
-proc makeBasicToml(dirName: string, path: string) {
-  const baseToml = '[brick]\n' +
-                     'name = "' + dirName + '"\n' +
-                     'version = "0.1.0"\n' +
-                     'chplVersion = "' + getChapelVersionStr() + '"\n' +
-                     '\n' +
-                     '[dependencies]' +
-                     '\n';
+proc getBaseTomlString(packageName: string, version: string, chapelVersion: string) {
+  const baseToml = """[brick]
+name = "%s"
+version = "%s"
+chplVersion = "%s"
+
+[dependencies]
+
```

Comment:
This adds an extra newline more than before, which shows up in the test `test/mason/chplVersion/mason-chpl-version-new.chpl`.

We either need to update the `.good` file or update this to remove one of the newlines.